man/networkd.conf.xml

```diff
@@ -61,6 +61,14 @@
         <listitem><para>Specifies the time interval to calculate the traffic speed of each interface.
         If <varname>SpeedMeter=no</varname>, the value is ignored. Defaults to 10sec.</para></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>IgnoreForeignRoutes=</varname></term>
```

Comment:
I'd like to invert the name, e.g., `ManageForeignRoutes=` and defaults to `yes`.